pulp_python/app/models.py

```diff
@@ -192,6 +192,9 @@ class PythonPackageContent(Content):
     packagetype = models.TextField(choices=PACKAGE_TYPES)
     python_version = models.TextField()
     sha256 = models.CharField(db_index=True, max_length=64)
+    sha256_metadata = models.CharField(max_length=64)
+    yanked = models.BooleanField(default=False)
```

Comment:
@gerrod3 So, I take it that the python plugin does not have "immutable" content, since "yanked" is explicitly mutable?

> The yanked attribute is not immutable once set, and may be rescinded in the future (and once rescinded, may be reset as well). Thus API users MUST be able to cope with a yanked file being “unyanked” (and even yanked again).